spec/test_app_templates/Gemfile.extra

```diff
@@ -1,4 +1,6 @@
 # placeholder to use for pinning against specific gem commit references
+gem 'hydra-pcdm', github:'projecthydra/hydra-pcdm', branch: 'master'
```

Comment:
Are these still needed? I'd like to see these taken out before we merge this PR -- and then we can cut the 2.0 release soon thereafter.